Mention that text fragments are less durable (#41178)
mdn/content: files/en-us/web/uri/reference/fragment/text_fragments/index.md

Reviewer: while here, can we also improve the intro para - the 1st sentence is long and convoluted and the flow of sentences is breaking up by the browser sentence.

> **Text fragments** let you link directly to specific text in a web page, without requiring the page author to add an ID. They use a special syntax in the URL fragment. This feature lets you create deep links to content you don't control, without relying on the presence of IDs. It also makes sharing links more useful, since you can directly point others to specific words. Browsers may differ in how they draw attention to the linked text (for example, with a color highlight) and/or scroll it into view.

</h2>
```

> Not all documents have such anchors, and even if they do, depending on the context, linking to a heading may be much less obvious than linking directly to the piece of text you are citing. This is where text fragments come in: it allows the link author to have full control over where in the document they want to link to, without requiring any special markup in the document itself. For example, a search engine may refer to a specific sentence in a page in its search results, and clicking the link will take you directly to that sentence.

Reviewer: - Let's bring back the deleted para to maintain continuity with the previous text. We can follow that up with your additions

> If the ID is changed or removed, the browser will ignore the fragment and will open the top of the page. This is reasonable in terms of graceful degradation, but it also means that you need to depend on the page's markup.

- Some suggestions to remove few words to make prose more compact
```suggestion
Not all documents have such anchors, and even if they do, linking to a heading might be much less obvious than linking directly to the specific text you are citing. This is where **text fragments** help: they allow the link author to have full control over what text to link to, without requiring any special markup in the target document. For example, a search engine may refer to a specific sentence in its search results, and clicking the link will take you directly to that sentence.
```

> Of course, text fragments also have the limitation that text in a document is less stable than document structure. If the linked document is updated, the fragment would not navigate to the correct point, but would stay at the top of the page. This is fine for transient links such as those in search results, but if you intend for the link to work over time, document fragments may be more reliable.

Reviewer: ```suggestion
However, text fragments also have a limitation: text in a document is less stable than document structure. If the text in the linked document is updated, the fragment will no longer match, and the browser will open the top of the page. This is fine for transient links such as those in search results, but if you intend for the link to work over time, document fragments may be more reliable.
```
